Neos.Flow/Classes/Cli/RequestBuilder.php

```diff
@@ -118,6 +118,14 @@ public function build($commandLine): Request
         $request = new Request();
         $request->setControllerObjectName(HelpCommandController::class);
 
+        // @TODO: Can this be implemented even better/cleaner?
```

Comment:
I don't really see how, AFAIK we do not have a concept for such global things, but I'd say whatever _next_ thing we want to implement that is global (-v(v(v)) maybe???) we can then think about a better structure.

Reply:
I agree